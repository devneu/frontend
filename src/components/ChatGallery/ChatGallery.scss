.chat-gallery {
  padding: 0 10px;

  .gallery-title {
    text-transform: uppercase;
  }

  .gallery-list {
    display: grid;
    grid-template-columns: repeat(3, 1fr);
    grid-auto-rows: 1fr;
    grid-gap: 5px;
  }

  .gallery-img {
    border-radius: 10px;
    object-fit: cover;
  }
}
