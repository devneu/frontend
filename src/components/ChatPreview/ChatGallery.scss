.photo-multimedia {
  padding: 10px;

  .media-list {
    display: grid;
    grid-template-columns: repeat(3, 1fr);
    grid-auto-rows: 1fr;
    grid-gap: 5px;
  }

  .gallery-img {
    height: 80px;
    max-width: 80px;
    border-radius: 20px;
    object-fit: cover;
  }
}
