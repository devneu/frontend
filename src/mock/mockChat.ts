export const photos = [
  {
    img: 'https://images.unsplash.com/photo-1469854523086-cc02fe5d8800?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8M3x8fGVufDB8fHx8&w=1000&q=80',
  },
  {
    img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSV_EfZmXNnuKrtvDecfz6B7r955J6X7EFTsw&usqp=CAU',
  },
  {
    img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSrOELmQBbTGeOYYBHmT6UxFVma2GOPE-6kpg&usqp=CAU',
  },
  {
    img: 'https://sourceduk.com/wp-content/uploads/2018/10/girlgoneabroad.jpg',
  },
  {
    img: 'https://www.workandtravelgroup.com/wp-content/uploads/wtg-group-2.jpg',
  },
  {
    img: 'https://www.moneycrashers.com/wp-content/uploads/2018/10/travel-world-map-1068x713.jpg',
  },
];

export const attachments = [
  {
    name: 'pdf',
    img: 'https://www.iconpacks.net/icons/2/free-pdf-icon-3385.png',
    description: 'how to create a case study',
  },
  {
    name: 'zip',
    img: 'https://www.iconpacks.net/icons/2/free-zip-icon-1519-thumb.png',
    description: 'Project Documents',
  },
  {
    name: 'word',
    img: 'https://www.pngkey.com/png/full/210-2104640_imagen-microsoft-office-365-word-logo.png ',
    description: 'Competitor Analysis Template',
  },
];

export const avatar = {
  img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
};

export const groupData = {
  img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
  name: 'Nasir Uddin',
  description: 'Ux Designer',
  icon: 'https://i.pinimg.com/originals/44/9d/d1/449dd1f38a8bd87b41f7b960db0fefff.png',
  memberImgs: [
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://www.w3schools.com/howto/img_avatar2.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://www.w3schools.com/howto/img_avatar.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://www.w3schools.com/w3images/avatar6.png',
    },
  ],
};
