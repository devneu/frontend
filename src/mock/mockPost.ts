export const createMockPost = (options = {} as any) => ({
  id: options.id || 999,
  img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
  title: 'DEV-17.com',
  date: 'September 23, 2010',
  text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
  articleText: `
    # I have odd cosmic thoughts every day
    
    For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.
    
    Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
  tag: ['web', 'js'],
});

export const createMockPosts = () => [
  {
    id: 1,
    img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
    title: 'PostPage Title',
    text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
    date: 'September 23, 2010',
    articleText: `
# I have odd cosmic thoughts every day

For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.

Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
    tag: ['Web Design', 'Freebies'],
  },
  {
    id: 2,
    img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
    title: 'PostPage Title',
    text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
    date: 'September 23, 2010',
    articleText: `
# I have odd cosmic thoughts every day

For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.

Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
    tag: ['Web Design', 'Freebies'],
  },
  {
    id: 3,
    img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
    title: 'PostPage Title',
    text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
    date: 'September 23, 2010',
    articleText: `
# I have odd cosmic thoughts every day

For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.

Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
    tag: ['Web Design', 'Freebies'],
  },
  {
    id: 4,
    img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
    title: 'PostPage Title',
    text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
    date: 'September 23, 2010',
    articleText: `
# I have odd cosmic thoughts every day

For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.

Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
    tag: ['Web Design', 'Freebies'],
  },
];
