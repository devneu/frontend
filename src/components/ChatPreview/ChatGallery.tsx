import React from 'react';
import Paragraph from 'antd/es/typography/Paragraph';
import { Button } from 'antd';
import './ChatGallery.scss';

const ChatGallery = ({ photos }: any) => (
  <div className="chat-gallery">
    <Paragraph>Photos & multimedia</Paragraph>
    <div className="gallery-list">
      {
        // eslint-disable-next-line react/no-array-index-key
        photos.map((photo: any) => (
          <img className="gallery-img" src={photo.img} key={photo.img} alt="#" />
        ))
      }
    </div>
    <Button className="show-all">View all</Button>
  </div>
);

export default ChatGallery;
