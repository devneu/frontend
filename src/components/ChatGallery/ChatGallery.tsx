import React from 'react';
import { Button, Typography, Image } from 'antd';
import './ChatGallery.scss';
import { Photos } from '../../types/ChatPreview.interface';

interface ChatGalleryProps {
  photos: Photos[];
}
const { Paragraph } = Typography;

const ChatGallery = ({ photos }: ChatGalleryProps) => (
  <div className="chat-gallery">
    <Paragraph className="gallery-title" strong>
      Photos & Multimedia
    </Paragraph>
    <div className="gallery-list">
      <Image.PreviewGroup>
        {photos.map((photo) => (
          <Image
            className="gallery-img"
            src={photo.img}
            key={photo.img}
            alt="#"
            height={55}
            width={55}
          />
        ))}
      </Image.PreviewGroup>
    </div>
    <Button className="show-all" type="link" block>
      View all
    </Button>
  </div>
);

export default ChatGallery;
