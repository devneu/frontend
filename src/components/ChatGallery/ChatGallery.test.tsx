import React from 'react';
import renderer from 'react-test-renderer';
import ChatGallery from './ChatGallery';
import { photos } from '../../mock/mockChat';

describe('ChatGallery', () => {
  it('Should successfully render', () => {
    const wrapper = renderer.create(<ChatGallery photos={photos} />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
