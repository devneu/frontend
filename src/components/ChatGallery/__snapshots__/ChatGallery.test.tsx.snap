// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatGallery Should successfully render 1`] = `
<div
  className="chat-gallery"
>
  <div
    className="ant-typography gallery-title"
    style={
      Object {
        "WebkitLineClamp": undefined,
      }
    }
  >
    <strong>
      Photos & Multimedia
    </strong>
  </div>
  <div
    className="gallery-list"
  >
    <div
      className="ant-image"
      onClick={[Function]}
      style={
        Object {
          "height": 55,
          "width": 55,
        }
      }
    >
      <img
        alt="#"
        className="ant-image-img gallery-img"
        onError={[Function]}
        onLoad={[Function]}
        src="https://images.unsplash.com/photo-1469854523086-cc02fe5d8800?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8M3x8fGVufDB8fHx8&w=1000&q=80"
        style={
          Object {
            "height": 55,
          }
        }
      />
      <div
        className="ant-image-mask"
      >
        <div
          className="ant-image-mask-info"
        >
          <span
            aria-label="eye"
            className="anticon anticon-eye"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="eye"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M942.2 486.2C847.4 286.5 704.1 186 512 186c-192.2 0-335.4 100.5-430.2 300.3a60.3 60.3 0 000 51.5C176.6 737.5 319.9 838 512 838c192.2 0 335.4-100.5 430.2-300.3 7.7-16.2 7.7-35 0-51.5zM512 766c-161.3 0-279.4-81.8-362.7-254C232.6 339.8 350.7 258 512 258c161.3 0 279.4 81.8 362.7 254C791.5 684.2 673.4 766 512 766zm-4-430c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm0 288c-61.9 0-112-50.1-112-112s50.1-112 112-112 112 50.1 112 112-50.1 112-112 112z"
              />
            </svg>
          </span>
          Preview
        </div>
      </div>
    </div>
    <div
      className="ant-image"
      onClick={[Function]}
      style={
        Object {
          "height": 55,
          "width": 55,
        }
      }
    >
      <img
        alt="#"
        className="ant-image-img gallery-img"
        onError={[Function]}
        onLoad={[Function]}
        src="https://lp-cms-production.imgix.net/features/2019/05/Solo-Travel-in-Nature-acbfea52bfaf.jpg"
        style={
          Object {
            "height": 55,
          }
        }
      />
      <div
        className="ant-image-mask"
      >
        <div
          className="ant-image-mask-info"
        >
          <span
            aria-label="eye"
            className="anticon anticon-eye"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="eye"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M942.2 486.2C847.4 286.5 704.1 186 512 186c-192.2 0-335.4 100.5-430.2 300.3a60.3 60.3 0 000 51.5C176.6 737.5 319.9 838 512 838c192.2 0 335.4-100.5 430.2-300.3 7.7-16.2 7.7-35 0-51.5zM512 766c-161.3 0-279.4-81.8-362.7-254C232.6 339.8 350.7 258 512 258c161.3 0 279.4 81.8 362.7 254C791.5 684.2 673.4 766 512 766zm-4-430c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm0 288c-61.9 0-112-50.1-112-112s50.1-112 112-112 112 50.1 112 112-50.1 112-112 112z"
              />
            </svg>
          </span>
          Preview
        </div>
      </div>
    </div>
    <div
      className="ant-image"
      onClick={[Function]}
      style={
        Object {
          "height": 55,
          "width": 55,
        }
      }
    >
      <img
        alt="#"
        className="ant-image-img gallery-img"
        onError={[Function]}
        onLoad={[Function]}
        src="https://www.workaway.info/gfx/2015/content/frontpage/header_frontpage_5.jpg"
        style={
          Object {
            "height": 55,
          }
        }
      />
      <div
        className="ant-image-mask"
      >
        <div
          className="ant-image-mask-info"
        >
          <span
            aria-label="eye"
            className="anticon anticon-eye"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="eye"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M942.2 486.2C847.4 286.5 704.1 186 512 186c-192.2 0-335.4 100.5-430.2 300.3a60.3 60.3 0 000 51.5C176.6 737.5 319.9 838 512 838c192.2 0 335.4-100.5 430.2-300.3 7.7-16.2 7.7-35 0-51.5zM512 766c-161.3 0-279.4-81.8-362.7-254C232.6 339.8 350.7 258 512 258c161.3 0 279.4 81.8 362.7 254C791.5 684.2 673.4 766 512 766zm-4-430c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm0 288c-61.9 0-112-50.1-112-112s50.1-112 112-112 112 50.1 112 112-50.1 112-112 112z"
              />
            </svg>
          </span>
          Preview
        </div>
      </div>
    </div>
    <div
      className="ant-image"
      onClick={[Function]}
      style={
        Object {
          "height": 55,
          "width": 55,
        }
      }
    >
      <img
        alt="#"
        className="ant-image-img gallery-img"
        onError={[Function]}
        onLoad={[Function]}
        src="https://sourceduk.com/wp-content/uploads/2018/10/girlgoneabroad.jpg"
        style={
          Object {
            "height": 55,
          }
        }
      />
      <div
        className="ant-image-mask"
      >
        <div
          className="ant-image-mask-info"
        >
          <span
            aria-label="eye"
            className="anticon anticon-eye"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="eye"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M942.2 486.2C847.4 286.5 704.1 186 512 186c-192.2 0-335.4 100.5-430.2 300.3a60.3 60.3 0 000 51.5C176.6 737.5 319.9 838 512 838c192.2 0 335.4-100.5 430.2-300.3 7.7-16.2 7.7-35 0-51.5zM512 766c-161.3 0-279.4-81.8-362.7-254C232.6 339.8 350.7 258 512 258c161.3 0 279.4 81.8 362.7 254C791.5 684.2 673.4 766 512 766zm-4-430c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm0 288c-61.9 0-112-50.1-112-112s50.1-112 112-112 112 50.1 112 112-50.1 112-112 112z"
              />
            </svg>
          </span>
          Preview
        </div>
      </div>
    </div>
    <div
      className="ant-image"
      onClick={[Function]}
      style={
        Object {
          "height": 55,
          "width": 55,
        }
      }
    >
      <img
        alt="#"
        className="ant-image-img gallery-img"
        onError={[Function]}
        onLoad={[Function]}
        src="https://www.workandtravelgroup.com/wp-content/uploads/wtg-group-2.jpg"
        style={
          Object {
            "height": 55,
          }
        }
      />
      <div
        className="ant-image-mask"
      >
        <div
          className="ant-image-mask-info"
        >
          <span
            aria-label="eye"
            className="anticon anticon-eye"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="eye"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M942.2 486.2C847.4 286.5 704.1 186 512 186c-192.2 0-335.4 100.5-430.2 300.3a60.3 60.3 0 000 51.5C176.6 737.5 319.9 838 512 838c192.2 0 335.4-100.5 430.2-300.3 7.7-16.2 7.7-35 0-51.5zM512 766c-161.3 0-279.4-81.8-362.7-254C232.6 339.8 350.7 258 512 258c161.3 0 279.4 81.8 362.7 254C791.5 684.2 673.4 766 512 766zm-4-430c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm0 288c-61.9 0-112-50.1-112-112s50.1-112 112-112 112 50.1 112 112-50.1 112-112 112z"
              />
            </svg>
          </span>
          Preview
        </div>
      </div>
    </div>
    <div
      className="ant-image"
      onClick={[Function]}
      style={
        Object {
          "height": 55,
          "width": 55,
        }
      }
    >
      <img
        alt="#"
        className="ant-image-img gallery-img"
        onError={[Function]}
        onLoad={[Function]}
        src="https://www.moneycrashers.com/wp-content/uploads/2018/10/travel-world-map-1068x713.jpg"
        style={
          Object {
            "height": 55,
          }
        }
      />
      <div
        className="ant-image-mask"
      >
        <div
          className="ant-image-mask-info"
        >
          <span
            aria-label="eye"
            className="anticon anticon-eye"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="eye"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M942.2 486.2C847.4 286.5 704.1 186 512 186c-192.2 0-335.4 100.5-430.2 300.3a60.3 60.3 0 000 51.5C176.6 737.5 319.9 838 512 838c192.2 0 335.4-100.5 430.2-300.3 7.7-16.2 7.7-35 0-51.5zM512 766c-161.3 0-279.4-81.8-362.7-254C232.6 339.8 350.7 258 512 258c161.3 0 279.4 81.8 362.7 254C791.5 684.2 673.4 766 512 766zm-4-430c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm0 288c-61.9 0-112-50.1-112-112s50.1-112 112-112 112 50.1 112 112-50.1 112-112 112z"
              />
            </svg>
          </span>
          Preview
        </div>
      </div>
    </div>
  </div>
  <button
    className="ant-btn ant-btn-link ant-btn-block show-all"
    onClick={[Function]}
    type="button"
  >
    <span>
      View all
    </span>
  </button>
</div>
`;
